src/Translation_unit.ml

```diff
@@ -91,7 +91,8 @@ let dump_formatted ~input_name ?output_file ~suffix fmted =
   else None
 
 let print_error ?(quiet_unstable = false) ?(quiet_comments = false)
-    ?(quiet_doc_comments = false) conf fmt input_name error =
+    ?(quiet_doc_comments = false) ?(fmt = Format.err_formatter) conf
+    ~input_name error =
```

Comment:
`quiet_unstable`, `quiet_comments` and `quiet_doc_comments` are never used. They can be removed and `quiet_exn` below can be simplified to `false`.

Reply:
done